tests/unit/account/test_workspaces.py

```diff
@@ -34,95 +34,35 @@ def mock_response(endpoint, **kwargs):
     return inner
 
 
-def test_subscriptions_name_to_id(arm_requests):
-    arm_requests(
-        {
-            "/subscriptions": {
-                "value": [
-                    {"displayName": "first", "subscriptionId": "001", "tenantId": "xxx"},
-                    {"displayName": "second", "subscriptionId": "002", "tenantId": "def_from_token"},
-                    {"displayName": "third", "subscriptionId": "003", "tenantId": "def_from_token"},
-                ]
-            }
-        }
-    )
-    cfg = Config(host="https://accounts.azuredatabricks.net", auth_type="azure-cli")
-
-    awl = AzureWorkspaceLister(cfg)
-    subs = awl.subscriptions_name_to_id()
-
-    assert {"second": "002", "third": "003"} == subs
-
-
-def test_list_azure_workspaces(arm_requests):
-    arm_requests(
-        {
-            "/subscriptions": {
-                "value": [
-                    {"displayName": "first", "subscriptionId": "001", "tenantId": "xxx"},
-                    {"displayName": "second", "subscriptionId": "002", "tenantId": "def_from_token"},
-                    {"displayName": "third", "subscriptionId": "003", "tenantId": "def_from_token"},
-                ]
-            },
-            "/subscriptions/002/providers/Microsoft.Databricks/workspaces": {
-                "value": [
-                    {
-                        "id": ".../resourceGroups/first-rg/...",
-                        "name": "first-workspace",
-                        "location": "eastus",
-                        "sku": {"name": "premium"},
-                        "properties": {
-                            "provisioningState": "Succeeded",
-                            "workspaceUrl": "adb-123.10.azuredatabricks.net",
-                            "workspaceId": "123",
-                        },
-                    },
-                    {
-                        "id": ".../resourceGroups/first-rg/...",
-                        "name": "second-workspace",
-                        "location": "eastus",
-                        "sku": {"name": "premium"},
-                        "properties": {
-                            "provisioningState": "Succeeded",
-                            "workspaceUrl": "adb-123.10.azuredatabricks.net",
-                            "workspaceId": "123",
-                        },
-                    },
-                ]
-            },
-        }
-    )
+@pytest.fixture()
+def workspace_mock(mocker):
```

Comment:
rename this to `account_workspaces_mock`. `workspace_mock` might mean something else.